tik/lib/include/tik/libtik.h

```diff
@@ -1,10 +1,11 @@
-#include "tik/Kernel.h"
+#pragma once
+#include "tik/CartographerKernel.h"
 #include <llvm/IR/Module.h>
 #include <map>
 
 namespace TraceAtlas::tik
 {
     extern llvm::Module *TikModule;
-    extern std::map<llvm::Function *, std::shared_ptr<Kernel>> KfMap;
-    extern std::map<int64_t, std::shared_ptr<Kernel>> KernelMap;
+    extern std::map<llvm::Function *, std::shared_ptr<CartographerKernel>> KfMap;
+    extern std::map<int64_t, std::shared_ptr<CartographerKernel>> KernelMap;
```

Comment:
Isn't this wrong? It should be Kernel I thought?

Reply:
I can make that happen. I changed this to CartographerKernel because I didn't migrate KernelImport and KernelExport to use ValueIDs, they still use the pointers themselves. I'll have to create a value map and pass it everywhere needed in the CartographerKernel constructor, then move these declaractions to the Kernel class again